index.js

```diff
@@ -80,9 +84,9 @@ const BluedotPointSDK = {
     unsubscribe,
     unsubscribeAll,
     setCustomEventMetaData,
+    getCustomEventMetaData,
```

Comment:
Was this planned for Android as well in SDK 16 release?

Reply:
yes